tutorials/git-basics.rst

```diff
@@ -0,0 +1,909 @@
+refer to as your *temp* repository.
+
+
+Now, switch to the window that is open to your home repository, add a
+line to ``test.txt`` with the text ``One more change!``. Create a
+commit for that change::
+
+    git commit -m"Adding one more change"
+
+And push it to GitHub (you should know how to
+do this by now, but make sure to ask for help if
+you’re unsure of how to proceed).
+
+Next, switch to the window that is open to your temp repository, check
+if that change appears in the ``test.txt`` file. It will not, because
+you have not yet downloaded the latest commits from the
+repository. You can do this by running this command:
+
+::
+
+   git pull
+
+This should output something like this:
+
+::
+
+    remote: Enumerating objects: 5, done.
+    remote: Counting objects: 100% (5/5), done.
+    remote: Compressing objects: 100% (2/2), done.
+    remote: Total 3 (delta 0), reused 3 (delta 0), pack-reused 0
+    Unpacking objects: 100% (3/3), 312 bytes | 20.00 KiB/s, done.
+    From git@github.com:GITHUB_USERNAME/uchicago-cs-git-tutorial.git
+       e3f9ef1..5716877  main       -> origin/main
+    Updating e3f9ef1..5716877
+    Fast-forward
+     test.txt | 3 ++-
+     1 file changed, 2 insertions(+), 1 deletion(-)
+
+
+If you have multiple local repositories (e.g., one on a CS machine and
+one on your laptop), it is very important that you remember to run
+``git pull`` before you start working, and that you ``git push`` any
+changes you make. Otherwise, your local repositories (and the repository
+on GitHub) may *diverge* leading to a messy situation called a *merge
+conflict* (we discuss conflicts in more detail below). This will be
+specially important once you start using Git for its intended purpose:
+to collaborate with multiple developers, where each developer will have
+their own local repository, and it will become easier for some
+developers’ code to diverge from others’.
+
+Discarding changes and unstaging
+--------------------------------
+
+One of the benefits of using a version control system is that it is very
+easy to inspect the history of changes to a given file, as well as to
+undo changes we did not intend to make.
+
+For example, edit ``test.txt`` to remove all its contents. Make sure you
+do this in your home repository (``/home/USER/git-tutorial/``)
+and not in the temp repository you created earlier.
+
+``git status`` will tell us this:
+
+::
+
+    On branch main
+    Your branch is up to date with 'origin/main'.
+
+    Changes not staged for commit:
+      (use "git add <file>..." to update what will be committed)
+      (use "git restore <file>..." to discard changes in working directory)
+        modified:   test.txt
+
+
+If we want to discard the changes we made to ``test.txt``, all we have
+to do is follow the helpful advice provided by the above output:
+
+::
+
+   git restore test.txt
+
+.. note::
+
+   In older versions of Git, ``git status`` may refer to the ``git checkout`` command.
```

Comment:
Did you meant `git restore` instead of `git status`?

Reply:
No, the note refers to the output from `git status` (which, in older versions, will tell you to run `git checkout` instead of `git restore`)